### PR TITLE
Ability to reopen savings accounts when closed or rejected - First stage

### PR DESCRIPTION
**Pull request by ippezshelby**

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java [hunk 1]
```diff
@@ -446,6 +446,72 @@ public CommandProcessingResult rejectApplication(final Long savingsId, final Jso
...
+                .withClientId(savingsAccount.clientId()) //
+                .withGroupId(savingsAccount.groupId()) //
+                .withSavingsId(savingsId) //
+                .with(changes) //
+                .build();
+    }
+
+    @Transactional
+    @Override
+    public CommandProcessingResult reopen(final Long savingsId, final JsonCommand command) {
```
Comment (nazeer1100126) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java:484]: While closing a savings account user can withdraw amount. There will be accounting entry in journal entry table if accounting is enabled for that savings product. If you are re opening this account, what happens to that entry?

Comment (ippezshelby) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java:484]: @nazeer1100126 , Actually that depends on what payment method has been used. For example, if the payment method is cash during account closure, then when the account is reopened the balance is zero as this directly affected cash. If payment method is not by cash (i.e by Voucher) then the balances at closure are to be booked to a GL account say "Account Closure Account" and when the account is reopened, the balance is got from the GL Account (Account Closure Account) and posted back to the account reopened.

Comment (nazeer1100126) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java:484]: I don't see those changes here. It just resets the account status or am I missing something?

Comment (ippezshelby) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsApplicationProcessWritePlatformServiceJpaRepositoryImpl.java:484]: @nazeer1100126 , These changes will only be possible if the issues https://issues.apache.org/jira/browse/FINERACT-232 and  https://issues.apache.org/jira/browse/FINERACT-408 are implemented and those are more technical for me.

**Comment by vorburger** [comment]
@Ippezrobert thank you very much for your interest in Mifos!  I'm an (old) volunteer who got reminded of Mifos this week-end at the FOSDEM conference, and I'm taking a moment to help the project clean up some of it's old issues and pull requests etc.  

We were wondering whether you are still alive, and would be willing to rebase this PR and take any previous review feedback into account?  If we don't hear from you within 2 weeks, we will take the liberty to close this PR, to avoid confusion for future new contributors looking at this repository - hope you understand and that's fair.

**Comment by vorburger** [comment]
Friendly service reminder: I will close this PR in 3 days to help this project have a clean list of active PRs.

**Pull request closed without merging**